pkg/api/validation/validation.go

```diff
@@ -2436,6 +2436,7 @@ func validateContainersOnlyForPod(containers []api.Container, fldPath *field.Pat
 		if len(ctr.Image) != len(strings.TrimSpace(ctr.Image)) {
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("image"), ctr.Image, "must not have leading or trailing whitespace"))
 		}
+		allErrs = append(allErrs, validateSecurityContext(ctr.SecurityContext)...)
```

Comment:
What is the reason to perform such validation in the `validateContainersOnlyForPod()` instead of within the existing `ValidateSecurityContext()`?

Reply:
nothing other than I couldn't find `ValidateSecurityContext`

Reply:
will move.